deployments/kustomize/overlays/openshift/images/base/api/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: api-deployment
+  annotations:
+    image.openshift.io/triggers: >-
```

Comment:
this trigger should just be removed, as it is specific to a namespace, and at this point everything should be namespace agnostic.